src/main/ruby/truffleruby/core/io.rb

```diff
@@ -1648,6 +1650,23 @@ def printf(fmt, *args)
     write sprintf(fmt, *args)
   end
 
+  def timeout
+    @timeout
+  end
```

Comment:
Let's use `attr_reader` here

Reply:
Done. Is that faster internally?

Reply:
Yes, that actually avoids the call (CallTarget/DirectCallNode) entirely :)